pagopa/io-app
# fix: [IOCOM-411] Fix overflowing text in notification preview in opt-in screen

## Short description
This PR fixes the overflowing text in the notification preview in the opt-in screen when the text is too long

### Preview
| Before | After |
| - | - |
|![Before](https://github.com/pagopa/io-app/assets/29163287/e9b060a2-67bf-4b40-81e2-8d72940e9043) |![After](https://github.com/pagopa/io-app/assets/29163287/29666b03-b404-4889-9fbf-0fbab1f962f7) |

## List of changes proposed in this pull request
See commit list

## How to test
Using the `io-dev-api-server`, run the application and go to the opt-in screen. Verify that the preview is displayed correctly.
